account_invoice_constraint_chronology/view/account_view.xml

```diff
@@ -1,15 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<!-- Copyright 2015-2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
     <data>
         <record id="view_account_journal_form" model="ir.ui.view">
             <field name="name">account.journal.form (account_constraint_chronology)</field>
             <field name="model">account.journal</field>
             <field name="inherit_id" ref="account.view_account_journal_form"/>
             <field name="arch" type="xml">
-                <field name="entry_posted" position="after">
+                <field name="refund_sequence" position="after">
                     <field name="check_chronology" attrs="{'readonly': [('type', 'not in', ['sale', 'purchase', 'sale_refund', 'purchase_refund'])]}"/>
                 </field>
              </field>
         </record>
     </data>
-</openerp>
```

Comment:
@GillesTephaneMeyomesse journal types 'sale_refund' and 'purchase_refund' [do not exist anymore](https://github.com/odoo/odoo/blob/635f6931d414d72d0a268f913d13c3722a71e3d5/addons/account/models/account.py#L235) in Odoo 10